mergify_engine/web/__init__.py

```diff
@@ -50,8 +50,12 @@
 @app.on_event("startup")
 async def startup():
```

Comment:
This is not typed, this is why mypy did not catch it.
Would you mind typing it?

Reply:
Ah thx!